### PR TITLE
Added/Fixed blocks 

### PR DESCRIPTION
**Pull request by BoredSathvik**
I fixed some blocks and added like 3 blocks ._.

### CODE REVIEW

File: packages/prismarine/src/block/blocks/Cobweb.ts [hunk 1]
```diff
@@ -5,14 +5,13 @@ import Flowable from '../Flowable';
...
         super({
-            name: 'minecraft:web', // Called cobweb in the Java Edition
+            name: 'minecraft:cobweb',
             id: BlockIdsType.Cobweb,
             hardness: 4
         });
     }
 
     public getToolType() {
-        // FIXME: add sword
```
Comment (filfat) [inline packages/prismarine/src/block/blocks/Cobweb.ts:15]: This isnt correct though. Both shears and sword is correct for cobweb.

Comment (BoredSathvik) [reply packages/prismarine/src/block/blocks/Cobweb.ts:15]: soo... how do i do it

Comment (Jakobg1215) [reply packages/prismarine/src/block/blocks/Cobweb.ts:15]: return {BlockToolType.Shears, BlockToolType.Sword}?

**Review by Jakobg1215 — APPROVED** [review]

File: packages/prismarine/src/block/blocks/Cobweb.ts (hunk 1 above)
Comment (filfat) [reply packages/prismarine/src/block/blocks/Cobweb.ts:15]: No, it’d have to be an array. But that isn’t supported yet. Keep it as it was before (with the TODO comment)

Comment (filfat) [reply packages/prismarine/src/block/blocks/Cobweb.ts:15]: @BoredSathvik just revert it the removal of the comment and we'll fix it in the future.

Comment (BoredSathvik) [reply packages/prismarine/src/block/blocks/Cobweb.ts:15]: ok

**Review by BoredSathvik — COMMENTED** [review]
Reverted

**Pull request merged**